lib/warnings.js

```diff
@@ -0,0 +1,20 @@
+var chalk = require('chalk')
+
+module.exports = {
+  deprecation: function (template, name) {
```

Comment:
Since we might want to have other kinds of deprecations in the future, maybe we can rename this warning to `v2SuffixTemplatesDeprecated`. I know it's a bit long, but I think that's fine since we're not using it constantly. It's also much clearer what's being deprecated.